res/database/Default/database_programmes.xml

```diff
@@ -11503,8 +11636,8 @@ Dream scenes and sentiments warp the storyline.</en>
 				<en>Mazes and Monsters</en>
 			</title_original>
 			<description>
-				<de>[1|Full] erster Film. Ein absurder Thriller über einen Rollenspieler der anfängt Obdachlose zu ermorden. Denn jeder weiß ja, Rollenspiele kommen von Satan.</de>
-				<en>[1|Full]'s first film. An absurd thriller about a roleplayer who starts murdering homeless people. Because everyone knows, role-playing games come from Satan himself.</en>
+				<de>[1|Full]' erster Film. Ein absurder Thriller über einen Rollenspieler, der anfängt, Obdachlose zu ermorden. Denn jeder weiß ja, Rollenspiele kommen von Satan.</de>
```

Comment:
Rather "Erster Film von..." in order to prevent problems with spoof/real name? (English as well)

Reply:
Yeah I'm aware of this and have an eye on that. General guideline should be, if a name ends with s, spoof name should also end with s (not that this is always followed). Here that is the case. But the rephrasing can be applied should this problem pop up elsewhere.